about.html

```diff
@@ -0,0 +1,149 @@
+                        <li class='nav-item'><a href="#partner">Sponsor</a></li>
+                        <li class='nav-item'><a href="about.html">About</a></li>
+                        <li class='nav-item'><a href="#" class="nav-btn">Ew campaign</a></li>
+                    </ul>
+                </div>
+                <div class="menu-btn">
+                    <div class="menu-btn__burger"></div>
+                </div>
+            </div>
+        </nav>
+    </div>
+
+    <!-- Main content -->
+    <div class="main">
+        <section class="landing">
+            <div class="container">
+                <div class="about-content">
+                    <h3 class="about-welcome-text">"Hello! Sharing world"</h3>
+                    <h1>THE VALUE OF <br> COLLABORATION IN<br>A REMOTE ENVIRONMENT.</h1>
+                    <p class="about-p">
+                        In particular, at this year's CC Summit, we
+                        are waiting for the participation of various organizations and individual participants who want
+                        to participate in activities to spread the value of sharing, free culture, and open knowledge
+                        with CC.
+                        Free software supporters, Wikipedia activists, open knowledge foundation activists, art
+                        galleries, libraries, museums, archives, governments,
+                        foundations, lawyers, social activists, etc. various people active in the field of open content
+                        and open knowledge gather in one place to
+                        share richer and more dynamic We hope that it will be an opportunity to share our thoughts on
+                        creating an ecosystem..
+                    </p>
+                    <p class="contact-message">Please contact us per
+                        Email for any further questions about Global Summit 2022</p>
+                    <a href="mailto:balixemma@gmail.com" class="Email-btn">balixemma@gmail.com <i
+                            class="fa-solid fa-chevron-right"></i></a>
+                </div>
+            </div>
+        </section>
+
+        <!--logo section -->
+        <section class="logo-exposition">
+            <div class="container">
+                <div class="logo-exposition-content">
+                    <h3 class="logo-exposition-title">Creative Commons Global Summit 2020 </h3>
+                    <p class="logo-exposition-p">
+                        A fun festival where people from more than 80 countries
+                        who believe in the value of sharing and openness and create
+                        positive change gather and share ideas will be held in Montreal in September.
+
+                    </p>
+                    <h1 class="logo-exposition-l logo">event<span>Work</span></h1>
+                </div>
+            </div>
+        </section>
+
+        <!-- history section -->
+        <section class="history">
```

Comment:
- [ ] Kindly add the partners sections after the past events section as provided in the design guidelines and as requested by the previous reviewer 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190827346-060129ba-0b58-42fe-8d69-8fce4431467c.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/190827370-5f05a820-df15-4407-98e5-5615823d766a.png)